module/os/linux/spl/spl-dcache.c

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (C) 2019 OpenZFS Project
+ *  Written by Pavel Snajdr <snajpa@snajpa.net>
+ *
```

Comment:
Rather than adding this at the spl layer I'd suggest we add this small function to `include/os/linux/kernel/linux/dcache_compat.h` as a `static inline`.  This header contains related dcache compatibility code for functions which have changed, or when we've needed a specific helper not provided by the kernel.